### DISCUSSION TITLE
Option to switch cameras?

### DISCUSSION THREAD

**Discussion by mitchellbunkers** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-js#251] [Q&A]
I'm running the example GitPage on an iPhone and a Samsung Galaxy S22, and after running Master, the front camera is the only option it will run. Is there a way to switch to the back facing camera?

**Comment by niyatim23** [comment]
Hi @mitchellbunkers, you need to configure to use the correct camera like it says [here](https://groups.google.com/g/discuss-webrtc/c/i07T1X_DY94/m/ZM6iuJkIAAAJ)

**Reply by mitchellbunkers** [reply]
Is it possible to add this to the test page so I can look at some example code? The link [here](https://groups.google.com/g/discuss-webrtc/c/i07T1X_DY94/m/ZM6iuJkIAAAJ) doesn't give clear examples on how to implement mediaDevices.

**Reply by sirknightj** [reply]
We can track that as a feature request. No ETA's at the moment.

To confirm, you did see the source code linked at the bottom of the other test page?
https://github.com/webrtc/samples/tree/gh-pages/src/content/devices/input-output/js
